wikimedia/wikipedia-ios
# Image recommendations - Connect edit summary and send feedback

**Phabricator:** https://phabricator.wikimedia.org/T358923

### Notes
This PR connects the edit summary view to the flow and uses the new canned messages. The flow is now connected end-to-end. I am also making the image recommendation feedback calls here upon successful edit publish and after the survey is answered for "No".

Since there's not a good testing environment, I am hiding the posting behind a feature flag, and setting that flag to false for the Experimental build. This is so design can test without worrying about updating live data.

### Test Steps
1. On Experimental, watch proxy for networking calls. Go through image recommendation flow. Confirm you do not see any post or put calls when answering the survey or publishing an edit.
2. Confirm canned messages work fine on Edit Summary.
3. On Staging, repeat step one. This time confirm edit and feedback calls go through, and you can see your edit on desktop.
4. Do a light regression test of the editor flow, (image insert and edit summary) on Test Wiki.
